events/court_events/court_events_general.txt

```diff
@@ -542,15 +542,17 @@ court.0108 = {
 		court_grandeur_current_level >= medium_court_grandeur_level
 	}
 	
+	# Warcraft
 	weight_multiplier = {
 		base = 1
 		modifier = {
 			add = 1
-			root.faith = {
-				OR = {
-					like_islam_religion_trigger = yes
-					has_doctrine_parameter = islamic_syncretic_actor_opinion_active
-				}
+			target = scope:foreign_prince.faith = {
+				has_faith = faith:cult_of_wealth
+				# OR = {
+					# like_islam_religion_trigger = yes
+					# has_doctrine_parameter = islamic_syncretic_actor_opinion_active
+				# }
```

Comment:
Syntax errors? Never seen such syntax in vanilla.

If it's supposed to check if `scope:foreign_prince.faith` is `faith:cult_of_wealth`, I believe should look like this: `scope:foreign_prince.faith = faith:cult_of_wealth`.

Reply:
I attempted this solution but it gives errors. Try searching `target = scope:` in \game\events` in the vanilla ck3 directory. There are quite a lot time where `target = scope:` points to a temporary scope defined first in that file. 

e.g. in `death_management_events.txt` there are these lines:
> [Line 8:] 	this = scope:dead_character
> [Line 1425] trigger = {	scope:dead_character.highest_held_title_tier >= root.highest_held_title_tier }

Reply:
> I attempted this solution but it gives errors.

Yeah, that makes sense. The scope may be not set in the modifier block. In the vanilla event, there is no scoping used.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/226206686-9e22e744-3e89-4488-92c8-06693d625e96.png)

</details>

> > [Line 1596] target = scope:dead_character

That's a part of the opinion = {} trigger syntax.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/226206785-9094059c-5197-47ef-863b-f2de95b9805e.png)

</details>

Reply:
Addressed in 7cc70ea8e